pkgs/shells/zsh/zplug/default.nix

```
@@ -0,0 +1,35 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  version = "2.3.4";
+  name = "zplug-${version}";
+
+  src = fetchFromGitHub {
+    owner = "zplug";
+    repo = "zplug";
+    rev = version;
+    sha256 = "1wcvk68fwyfba3kcin6fl0ifwf9mr8r914kq132nk96ga625j8b1";
+  };
+
+  phases = "installPhase";
+
+  installPhase = ''
+  mkdir -p $out/opt
+  cp -r $src $out/opt/zplug
+  '';
+
+  meta = with stdenv.lib; {
+    description = "A next-generation plugin manager for zsh.";
```

```suggestion
    description = "A next-generation plugin manager for zsh";
```

Please remove dot at the end of the description.

---

Got it.